pkg/ccl/streamingccl/streamclient/client_test.go

```diff
@@ -95,29 +92,3 @@ func ExampleClient() {
 	// "key_1"->value_1@1
 	// resolved 100
 }
-
-// Ensure that all implementations specified in this test properly close the
-// eventChannel when the given context is canceled.
-func TestImplementationsCloseChannel(t *testing.T) {
```

Comment:
This test wasn't really testing much, and we'll catch goroutine leaks in other tests, so removing.